changelog/unreleased/bugfix-stuck-after-session-expired

```diff
@@ -0,0 +1,9 @@
+Bugfix: Stuck After Session Expired
+
+We've fixed exit link to redirect to login once session expires
+
+We've removed the logout click handler and created a new logout component
+
+https://github.com/owncloud/web/issues/7453
+
```

Comment:
Please remove the blank line. The `calens` tool which we're using as changelog generator needs URLs at the end of the changelog file to follow each other without empty lines in between.